### DISCUSSION TITLE
how to make the bot reply to quotes in a group?

### DISCUSSION THREAD

**Discussion by kamil-malinski** [n3d1117/chatgpt-telegram-bot#98] [Q&A] [answered]
Hi!

I have disabled privacy mode and put "@botname" to "GROUP_TRIGGER_KEYWORD". This works as intended: the bot only replies to messages starting with "@botname".

Additionally I would like the bot to reply to replies to its own messages (when somebody quotes the bot's message). How can I achieve that?

Thanks!

Cheers

Kamil

**Comment by n3d1117** [comment, marked as answer]
Hi @kamil-malinski! Good idea, it should be enough to replace [these lines](https://github.com/n3d1117/chatgpt-telegram-bot/blob/7e6720f657eb67fa839d8877e77baccab00c2975/bot/telegram_bot.py#L298-L299):

https://github.com/n3d1117/chatgpt-telegram-bot/blob/7e6720f657eb67fa839d8877e77baccab00c2975/bot/telegram_bot.py#L298-L299

with:

```python
if update.message.reply_to_message and update.message.reply_to_message.from_user.id == context.bot.id:
    logging.info('Message is a reply to the bot, allowing...')
else:
    logging.warning('Message does not start with trigger keyword, ignoring...')
    return
```

Let me know if it works and i'll merge this!

**Reply by kamil-malinski** [reply]
works great, thanks a lot!

here's the push-request: https://github.com/n3d1117/chatgpt-telegram-bot/pull/99
